tools/lang_batch_translate.py

```
@@ -0,0 +1,192 @@
+    def walk(n):
+        if isinstance(n, dict):
+            for k, v in n.items():
+                if isinstance(v, str):
+                    add(v)
+                    if k == "pattern":
+                        for m in CHINESE_RUN.findall(v):
+                            add(m)
+                else:
+                    walk(v)
+        elif isinstance(n, list):
+            for x in n: walk(x)
```

Ruff (0.15.14)

[error] Multiple statements on one line (colon)

(E701)